src/docs/introduction/why-stencil.md

```diff
@@ -4,39 +4,38 @@ description: Stencil has a number of add-ons that you can use with the build pro
 url: /docs/introduction
 contributors:
   - jthoms1
+  - splitinfinities
+  - a-giuliano
 ---
 
-# Stencil: A Compiler for Web Components and High Performance Web Apps
+# Overview
 
-Stencil is a compiler that generates Web Components (more specifically, Custom Elements) and builds high performance web apps. Stencil combines the best concepts of the most popular frameworks into a simple build-time tool.
+## Stencil: A Web Components Compiler
 
-Stencil takes features such as
+Stencil is a compiler that generates Web Components (more specifically, Custom Elements). Stencil combines the best concepts of the most popular frameworks into a simple build-time tool.
 
-- Virtual DOM
-- Async rendering (inspired by React Fiber)
-- Reactive data-binding
-- TypeScript
-- JSX
-- Static Site Generation (SSG)
+Stencil uses TypeScript, JSX, and CSS to create standards-based Web Components that can be used to craft high quality component libraries, design systems, and applications.
 
-and then generates standards-based Web Components and web apps with these features baked in.
+Stencil generates standards-compliant Web Components that can work with popular frameworks right out of the box. In addition, Stencil can be used to generate framework native components that can be used just like any other components in your framework of choice. Stencil accomplishes this by wrapping your Web Components via Stencil's Output Target feature.
 
-Since Stencil generates standards-compliant web components, they can work with many popular frameworks right out of the box, and can be used without a framework because they are just web components. Stencil also enables a number of key capabilities on top of Web Components, in particular, prerendering, and objects-as-properties (instead of just strings).
-
-Compared to using Custom Elements directly, Stencil provides extra APIs that makes writing fast components simpler. APIs like Virtual DOM, JSX, and async rendering make fast, powerful components easy to create, while still maintaining 100% compatibility with Web Components.
+Compared to using Custom Elements directly, Stencil provides extra APIs that makes writing fast components simpler. APIs like Virtual DOM, JSX, and async rendering make fast, powerful components easy to create, while still maintaining 100% compatibility with Web Components. Stencil also enables a number of key capabilities on top of Web Components, in particular, prerendering, and objects-as-properties (instead of just strings).
```

Comment:
Should we link to the component API page here?

```suggestion
Compared to using Custom Elements directly, Stencil provides [extra APIs](https://stenciljs.com/docs/apis) that makes writing fast components simpler. APIs like Virtual DOM, JSX, and async rendering make fast, powerful components easy to create, while still maintaining 100% compatibility with Web Components. Stencil also enables a number of key capabilities on top of Web Components, in particular, prerendering, and objects-as-properties (instead of just strings).
```